backend/README.md

```diff
@@ -142,7 +142,7 @@ namespace Pims.Api.Migrations
 
 Any script that you want to run must be additionally included as build content in the project file.
 
-Edit the `Pims.Api.csproj` file and add the `<Content>` with a `<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>` (see below example).
+Edit the `pims.dal.csproj` file and add the `<Content>` with a `<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>` (see below example).
```

Comment:
Probably should keep the project names Pascal Case.

Reply:
The project name was originally renamed to all lower case. Agree we should go Pascal Case. Please note the the docker file will need to be updated (Linux is case sensitive)